etc/poco/bundle/main-test.json

```diff
@@ -96,14 +96,6 @@
 			"principals": [],
 			"allowed": true
 		},
-		{
```

Comment:
I've change to use the `/public/assets` to load those svgs, as those are assets from public folder, it make more sense to name it like that.